test/commands/app/delete/event.test.js

```diff
@@ -45,6 +31,10 @@ describe('Command Prototype', () => {
   })
 })
 
+// Question? What is/are the actual difference between this call and `delete action`?
```

Comment:
for now add/delete event adds/deletes an action but in the future it might add some other resources or create/delete resources in the backend
That's a choice made back some time ago with the events team